src/site/markdown/server/manage-webadmin.md

```diff
@@ -1545,6 +1545,49 @@ Response codes:
  - 400: The `fromDomain` resource name is invalid
  - 400: The destination domain specified in the body is invalid
 
+## Creating address regex
+You can use **webadmin** to create regex mappings.
+
+When a request containing a JSON body with two fields `source` and `regex` is sent, the route will update the database
+
+For example: send a request with body
+```
+{
+    "source": "james@domain.tld",
+    "regex": "^[aeiou]"
+}
+```
+the route will add a source `james@domain.tld` with a regex mapping `^[aeiou]` into RecipientRewriteTable.
+
+This feature uses [Recipients rewrite table](/server/config-recipientrewritetable.html) and requires
+the [RecipientRewriteTable API](https://github.com/apache/james-project/blob/master/server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/RecipientRewriteTable.java)
+to be configured.
+
+ - [Adding a regex mapping](#Adding_a_regex_mapping)
+ 
+### Adding a regex mapping
+```
+curl --header "Content-Type: application/json" \
+  --request POST \
+  --data '{"source":"james@domain.tld","regex":"^[aeiou]"}' \
+  http://ip:port/mappings/regex
```

Comment:
why did you choose this route? It looks not very consistent with everything else